wiki/Tournaments/CbC/2022/en.md

```diff
@@ -0,0 +1,177 @@
+---
+tags:
+  - Combro
+  - draft
+  - CbC
+  - Combro Cup
+  - Combro Cup Draft
+---
+
+# Combro Cup Banner
+
+![Combro Cup Banner](img/CbC-banner.png)
+
+The **Combro Cup** is a 3v3 captain draft-based osu!mania 4-key tournament hosted by ::{ flag=US }:: [FullCombro](https://osu.ppy.sh/users/14128407).
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-10-01/2022-10-22 (23:59 UTC-4) |
+| Qualifier showcase | 2022-10-30 (23:59 UTC-4) |
+| Qualifiers | 2022-11-01/2022-11-04 |
+| Drafting phase | 2022-11-05/2022-11-06 |
+| Round of 32 | 2022-11-11/2022-11-13 |
+| Round of 16 | 2022-11-18/2022-11-20 |
+| Quarterfinals | 2022-11-25/2022-11-27 |
+| Semifinals | 2022-12-02/2022-12-04 |
+| Finals | 2022-12-09/2022-12-11 |
+| Grand Finals | 2022-12-16/2022-12-18 |
+
+## Prizes
+
+*Prize Pool will be increased by community donations, currently is $1227 (2022-09-19)*
+
+| Placing | Prizes |
+| :-----: | :----- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 70% of the prize pool (*$858*) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the prize pool (*$245*) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10% of the prize pool (*$122*) |
```

Comment:
```suggestion
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 70% of the prize pool |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the prize pool |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 10% of the prize pool |
```

since the prize pool is ever-changing, i don't think it's necessary to include the exact amounts when we have the percentages